code/datums/quirks/negative.dm

```diff
@@ -894,3 +894,12 @@
 		return TRUE
 
 	return FALSE
+
+/datum/quirk/illiterate
+	name = "Illiterate"
+	desc = "You dropped out of school and are unable to read or write. This affects reading, writing, using computers and other electronics."
+	icon = "graduation-cap"
+	value = -8
+	mob_trait = TRAIT_ILLITERATE
+	medical_record_text = "Patient has a low IQ and is not literate."
```

Comment:
This could probably be put differently, better to relate the issue to specific brain trauma (or just explain aphasia) rather than just putting a dunce cap on someone.

Reply:
Yeah, what if I want to be a high IQ illiterate?

Reply:
Changed with b3bb575b1f343ea6b006ddf48b9120ad6b171415